bucket/sqlcl.json

```
@@ -0,0 +1,23 @@
+{
+    "version": "22.2.0.172.1718",
+    "description": "Oracle SQL Developer Command Line (SQLcl)",
+    "homepage": "https://www.oracle.com/sqlcl",
+    "license": {
+        "identifier": "Oracle Free Use License",
+        "url": "https://www.oracle.com/downloads/licenses/oracle-free-license.html"
+    },
+    "suggest": {
+        "JRE": "Java 11+"
```

```suggestion
        "JRE": "java/openjdk"
```

This needs to be an actual package name